changelog_unreleased/handlebars/11524.md

```diff
@@ -0,0 +1,45 @@
+#### Uses the opposite quote type for quotes inside mustache statements in attributes (#11524 by @bmaehr)
+
+<!-- prettier-ignore -->
+```hbs
+{{!-- Input --}}
+<div title="{{t 'login.username.description'}}" />
+
+{{!-- Prettier main --}}
+<div title="{{t 'login.username.description'}}" />
+
+{{!-- Prettier stable --}}
+<div title="{{t "login.username.description"}}" />
+
+```
+
+<!-- prettier-ignore -->
+```hbs
```

Comment:
I'm not sure it is necessary to have a second example. You could sophisticate slightly the first one to:
```hbs
<div title="foo {{t 'login.username.description'}}" />
```

Reply:
My idea for the first sample was the usual usage and the second one was one of the existing tests where the behavior really changed. Should I change it the way you suggested even if you already approved it?
I still have the message "First-time contributors need a maintainer to approve running workflows" - Could you do/handle this? Are there any further steps necessary from my side? Sorry for the questions but it is my first PR for this project.

Reply:
I am not a maintainer ;)

Reply:
The goal of having this changelog entry is to present the users with the recent changes. So I guess the more concise and precise the changelog is, the best!

Reply:
Ok, I removed the second example. Adding foo to the first one ist not necessary because from the user perspective only the change is interesting - the user expects that it works with simple and with complex cases and it is not necessary to some kind of "proove" that it also works for complex cases in the changelog (other than in a test).